apps/win32/nuget/Microsoft.FluentUI.FluentTesterWin32.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.FluentUI.FluentTesterWin32</id>
+    <version>1.0.0</version>
+    <title>FluentUI-React-Native Win32 Bundle</title>
+    <authors>Microsoft Office CXE</authors>
+    <projectUrl>https://github.com/microsoft/fluentui-react-native.git</projectUrl>
+    <description>This package contains the React Native JS bundle of FluentUI-React-Native's Win32 Test App.</description>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+  <files>
+    <file src="..\dist\index.win32.bundle" target="win32" />
```

Comment:
Generally things like `.nuspec` / `.podspec` / `package.json` files are at the root of the repo, but I suppose this is fine?

Reply:
I looked at a bunch of examples in fabric-internal and other standalone feature repos. I found they were usually in specific directories. I'm assuming both ways are ok :)

Reply:
Ah, good point! Forgot about fabric-internal 😆